ha/const.py

```diff
@@ -25,9 +25,10 @@
 PCSD_LOG="/var/log/pcsd/pcsd.log"
 HA_CMDS_OUTPUT="{}/ha_cmds_output".format(RA_LOG_DIR)
 COROSYNC_LOG="/var/log/cluster"
+ETC_HA_DIR="/etc/cortx/ha/"
```

Comment:
I believe this path is already defined by CONFIG_DIR

Reply:
Removed the duplicate path